tests/integration/targets/route53/tasks/main.yml

```diff
@@ -0,0 +1,1126 @@
+# tasks file for Route53 integration tests
+
+- set_fact:
+    zone_one: '{{ resource_prefix | replace("-", "") }}.one.ansible.test.'
+    zone_two: '{{ resource_prefix | replace("-", "") }}.two.ansible.test.'
+- debug:
+    msg: Set zones {{ zone_one }} and {{ zone_two }}
+
+- name: Test basics (new zone, A and AAAA records)
+  module_defaults:
+    group/aws:
+      aws_access_key: '{{ aws_access_key }}'
+      aws_secret_key: '{{ aws_secret_key }}'
+      security_token: '{{ security_token | default(omit) }}'
+      region: '{{ aws_region }}'
+    route53:
+      # Route53 is explicitly a global service
+      region:
```

Comment:
This would be a change in behaviour.  We've had this in place for a long time and this was working fine at the start of August: https://e2153ed399c2e1f53cbf-89091410d10a76c1df72c0776cd23562.ssl.cf1.rackcdn.com/1389/52a820470a884a77c6e7614e5663d467c2147928/gate/integration-community.aws-1/40b7ed1/job-output.txt

Reply:
Hmm, I think I am wrong. It does work to just unset, but I have to use the FQCN in the module defaults. Locally, if I just change it to `amazon.aws.route53` it works. I still don't know why that worked for community, though.

Reply:
My guess would be that this is related to the test environment that this is running in having *both* community.aws.route53 and amazon.aws.route53 and something weird happening with the way this is interacting.